src/app/search/search.component.ts

```diff
@@ -366,8 +415,74 @@ export class SearchComponent implements OnInit, OnDestroy {
       this.filters
     );
     this.resetEntryOrder();
-    this.updateSideBar(sideBarQuery);
     this.updateResultsTable(tableQuery);
+    this.updateSideBar(sideBarQuery);
+  }
+
+  // Searches for an author
+  // TODO: right now only returns exact matches, is case sensitive. Can be improved with new ES features when updated
```

Comment:
Thanks for the heads-up, I agree that a more sophisticated approach can wait for the ES update